src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/Selection.test.js

```diff
@@ -1211,15 +1228,98 @@ test('Should pass props with schema-options type correctly to MultiAutoComplete
         allowAdd: false,
         disabled: true,
         displayProperty: 'name',
-        filterParameter: 'names',
         idProperty: 'uuid',
-        locale,
-        resourceKey: 'snippets',
         searchProperties: ['name'],
-        value,
+        selectionStore: selection.instance().autoCompleteSelectionStore,
     }));
 });
 
+test('Should trigger a reload of the items if the value prop changes', () => {
```

Comment:
would be nice to include `auto_complete` somewhere here because this test only tests the `auto_complete` type. this is kind of confusing in this class in general. will probably get better when we split up the class in the future 🙂